src/Contracts/Data/Conversion.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 2.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Contracts\Data;
+
+
+interface Conversion
+{
+    /**
+     * Convert arrays to Object.
+     *
+     * @param array $array Arrays
+     *
+     * @since 2.0.0
+     *
+     * @return object
+     */
+    public static function arrayObject($array);
```

Comment:
To be consistency of `function naming`, we should use the `arrayToObject` method.

Reply:
Yes you're right